app/controllers/concerns/import.rb

```diff
@@ -0,0 +1,64 @@
+module Import
+  
+  def upload_page
+  end
+
+  def home
+    @histories_array = []
+    if current_offeror.history != nil
```

Comment:
could `Offeror.home_set` be rewritten so that it returns a `[]` if it receives a nil?  or we could wrap `current_offeror.history` in `Array()`?

Lot's of ideas from Avdi Grimm's confident ruby relevant here ...